mmv1/products/cloudrunv2/api.yaml

```
@@ -1015,6 +1015,22 @@ objects:
                             name: port
                             description: |-
                               Port number to access on the container. Must be in the range 1 to 65535. If not specified, defaults to 8080.
+                    - !ruby/object:Api::Type::NestedObject
+                      name: grpc
+                      description: |-
+                        GRPC specifies an action involving a GRPC port.
+                      properties:
+                        - !ruby/object:Api::Type::Integer
+                          name: port
+                          required: true
+                          description: |-
+                            Port number to access on the container. Number must be in the range 1 to 65535. If not specified, defaults to 8080.
```

Could you remove `If not specified, defaults to 8080.`? It's not accurate. The cloud run API documents will be updated as well.

Also appreciated if you can update the description for other probe ports.

---

@yanweiguo 

> Could you remove If not specified, defaults to 8080.? It's not accurate. The cloud run API documents will be updated as well.

Noted. Then, what would be the correct behavior of the Cloud Run Admin API? How about the revised description below?
```suggestion
                            Port number to access on the container. Number must be in the range 1 to 65535. If not specified, defaults to the port number that the gRPC service is actually listening on.
```


> Also appreciated if you can update the description for other probe ports.

Although understandable, that is beyond the scope of this pull request. Could you address the rest part in your separate pull request?

---

It defaults to the container port, which is the value of `container.ports[0].containerPort`.

It's not the port gRPC service is actually listening on since we don't know the actual behavior inside the container.